QualityControl/public/services/Layout.service.js

```diff
@@ -37,22 +37,29 @@ export default class LayoutService {
   }
 
   /**
-   * Method to get all layouts shared between users
+   * Method to get all layout cards shared between users
    * @param {Class<Observable>} that - Observer requesting data that should be notified of changes
    * @returns {undefined}
    */
-  async getLayouts(that = this.model) {
+  async getLayoutCards(that = this.model) {
     this.list = RemoteData.loading();
     that.notify();
+    const cardfields = 'id,name,owner_id,owner_name,description,isOfficial';
```

Comment:
The same goes in here, rather then `cardFields`, perhaps something like `requestedFields` or `fieldsToRetrieve`, etc.